src/libraries/Common/src/Interop/Windows/SspiCli/Interop.SSPI.cs

```diff
@@ -475,7 +475,7 @@ internal unsafe struct SecPkgCred_ClientCertPolicy
             string password,
             out SafeSspiAuthDataHandle authData);
 
-        // TODO: Switch to use GeneratedDllImport once we annotate blittable types used in interop in CoreLib (like Guid)
+        // TODO: [DllImportGenerator] Switch to use GeneratedDllImport once we annotate blittable types used in interop in CoreLib (like Guid)
```

Comment:
Might an issue be better than a TODO comment?

Reply:
I put it as a task in our tracking issue https://github.com/dotnet/runtime/issues/60595 with a note that we are annotating places with this TODO.